src/compas_occ/brep/__init__.py

```diff
@@ -19,8 +19,25 @@
     BRepFace
 
 """
+from compas.plugins import plugin
+
 from .brepvertex import BRepVertex  # noqa: F401
 from .brepedge import BRepEdge  # noqa: F401
 from .breploop import BRepLoop  # noqa: F401
 from .brepface import BRepFace  # noqa: F401
 from .brep import BRep  # noqa: F401
+
+
+@plugin(category="factories", requires=["OCC"])
+def new_brep(*args, **kwargs):
+    return object.__new__(BRep)
+
+
+@plugin(category="factories", requires=["OCC"])
+def from_brep(*args, **kwargs):
+    return BRep.from_shape(*args, **kwargs)
```

Comment:
without knowing in detail, these feels like a bit of a mismatch that `from_brep` is done using `from_shape`. Is it related to the known issues described in this PR?

Reply:
Yea this should be renamed. The original constructor was copied from here as `from_shape` but we decided on `from_brep` in core.